packages/react-components/react-nav-preview/stories/NavDrawerBody/index.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { NavDrawerBody } from '@fluentui/react-nav-preview';
 import descriptionMd from './NavDrawerBodyDescription.md';
 import bestPracticesMd from './NavDrawerBodyBestPractices.md';
 
-export { Default } from './NavDrawerBodyDefault.stories';
+// export { Default } from './NavDrawerBodyDefault.stories';
```

Comment:
this seems like leftover from previous PR ? 

also why commenting out the stories public api that is used for public docs ?

Reply:
These are all small, recomposed sub components that I don't want as part of the publicly facing documentation yet.

Reply:
Why not delete the stories instead? if they are not used in the docsite, they should be deleted.

Reply:
I'd rather keep them for when I need to work on the components in an isolated environment.

Reply:
commented out code = anti pattern, if you don't wanna have this within official public docsite please remove those exports. ty